demisto_sdk/commands/content_graph/tests/test_data/indicator_type.json

```diff
@@ -8,7 +8,7 @@
     "shouldPublish": false,
     "shouldCommit": false,
     "regex": "(?i)((?:(?:https?|ftps?|hxxps?|sftp|meows):((\\/\\/)|(\\\\\\\\))(www\\[?\\.\\]?)?|ftp\\[?\\.\\]?))(?:[-_\\d\\p{L}\\p{S}]+\\[?\\.\\]?)+[-_\\d\\p{L}\\[\\]]+(?::\\d+)?(?:(?:((\\/)|(\\*)|(\\#)|(\\?)))+(?:[-\\d\\p{L}+&@'#\\/%=~_$!\\-:.;\\\\*–—―|\\[\\]]*|\\([-\\d\\p{L}\\+&@#\\/%=~_\\$\\?!\\-:,;\\\\\\*–—―\\|\\[\\]]*\\))*)*|((?i)((((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.)+(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))|(([\\[\\]\\p{L}\\dֿ]+\\[?\\.\\]?){1,3}[\\p{L}]+))(:[\\d]+)?\\/([-\\p{L}\\+&@#\\/%=~_\\$\\?!\\-:,;\\\\\\*–—―\\|\\.]+([\\d]*)?)+)|((?:(?:https?|ftps?|hxxps?|sftp):((\\/\\/)|(\\\\\\\\)))[\\d][\\S]+)|(((https?|ftps?|hxxps?|sftp):((\\/\\/)|(\\\\\\\\)))?(\\[)?(([0-9a-fA-F]{1,4}:){7,7}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,7}:|([0-9a-fA-F]{1,4}:){1,6}:[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,5}(:[0-9a-fA-F]{1,4}){1,2}|([0-9a-fA-F]{1,4}:){1,4}(:[0-9a-fA-F]{1,4}){1,3}|([0-9a-fA-F]{1,4}:){1,3}(:[0-9a-fA-F]{1,4}){1,4}|([0-9a-fA-F]{1,4}:){1,2}(:[0-9a-fA-F]{1,4}){1,5}|[0-9a-fA-F]{1,4}:((:[0-9a-fA-F]{1,4}){1,6})|:((:[0-9a-fA-F]{1,4}){1,7}|:)|fe80:(:[0-9a-fA-F]{0,4}){0,4}%[0-9a-zA-Z]{1,}|::(ffff(:0{1,4}){0,1}:){0,1}((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])|([0-9a-fA-F]{1,4}:){1,4}:((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))(\\])?(:[\\d]+)?(((\\/|\\\\)[\\d]+(\\/|\\\\)[-\\d\\p{L}+&@'#\\/%=~_$!\\-:.;\\\\*–—―|\\[\\]]+)|((\\/|\\\\)[^\\d \\n\\t]+)))",
```

Comment:
lets check this out why this isnt failing at the moment

Reply:
The original validation only runs on new files